visual-testing-app/webpack.config.js

```diff
@@ -22,18 +22,40 @@ if (!info) {
 }
 
 module.exports = {
-  mode: 'development',
+  target: 'web',
+  mode: 'production',
```

Comment:
this doesn't seem to work with Percy. It might be a bug to do with percy?

When I run `yarn percy` locally after having built the project using mode: 'production', I get unstyled js in css, as shown [here](https://percy.io/commercetools-GmbH/ui-kit/builds/1290053). I even tried adding a wait for 2 seconds in case it was some sort of flash of unstyled content problem, and still got the same problem. When I removed back to development, it worked again as shown [here](https://percy.io/commercetools-GmbH/ui-kit/builds/1290049)

Reply:
I got an email back from Percy support. The gist of it is 

>  It looks like emotion uses the CSSOM (CSS Object Model) to inject its styles. Percy currently works by taking a snapshot of the DOM and the external assets linked in that DOM snapshot. So, at the moment, we're missing things in the CSSOM because it's not serialized into the DOM.


So it's not worth trying to solve why we can't use the webpack production build with emotion at this moment.

Reply:
So would it work if we extract the CSS into a file and use that? This would also help with using ui-kit for SSR.

Regarding SSR, it's still not an ideal solution, as we only want to inject the "critical styles" when rendering the page, but it might be a good starting point.
Anyway, this is a bit off topic...

Reply:
I think for UIKit as a library it's impossible to serve consumers a static `.css` file while fulfilling a potential requirement for only include above the fold css (or critical styles). Also if a static file was offered, and consumers include/use it, UIKit would have to stop inlining styles. In that it would have to know it the static css is/was used. At least in my head it's like a dog catching its tail. 

I think going all in on emotion, also for SRR and static extraction of css, is a good goal but doesn't really relate to Percy or what we're attempting to solve here.